packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-number/gux-form-field-number.tsx

```diff
@@ -158,6 +158,12 @@ export class GuxFormFieldNumber {
     }
   }
 
+  private focusInputElement(): void {
+    if (this.input) {
+      this.input.focus();
+    }
+  }
```

Comment:
If we do agree to this change, I will probably make a util out of this as it needs to be used in 2-3 form-fields.